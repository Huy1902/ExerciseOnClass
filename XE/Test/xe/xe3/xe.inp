50 100
15808 26671 20713 8107 14677 359 19600 1969 16822 2637 10028 4278 23678 14716 6172 11009 17236 15978 11889 9193 20636 16035 9060 14208 16810 24437 2417 9850 12891 11599 19527 18357 28450 15369 19992 3418 11800 15138 19710 13516 22888 26169 2034 21766 14898 10962 1651 26399 17039 28557 
30242 1859 19409 26848 18442 23690 2707 12101 350 24288 11303 5130 29284 4995 9860 14634 14665 11870 24307 352 18422 31933 28063 9271 24785 6993 21641 29236 9273 18772 12531 19481 24844 26524 29331 1841 18988 1225 12184 28884 29531 14530 5918 1424 21745 15741 31387 23154 2620 20200 24859 23097 12068 11608 24272 3780 28289 1472 4886 10201 8410 20617 26036 9064 30151 18075 13694 31608 4375 20942 27093 10306 18479 31556 11796 3823 17701 1444 28316 16638 5681 9065 18657 6139 250 13837 4098 8493 11302 26808 9112 17229 15595 18076 107 15138 1674 13839 5050 23771 