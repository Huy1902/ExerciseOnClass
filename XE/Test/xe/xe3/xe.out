3604458102
43 9 36 53 64 17 56 42 14 67 11 39 78 88 8 72 69 81 29 75 2 12 18 82 99 44 86 54 24 60 76 87 85 84 49 31 15 26 7 91 58 38 98 97 61 89 16 20 59 95 